src/components/cryptoList/index.tsx

```diff
@@ -0,0 +1,55 @@
+import React from 'react';
+import {
+  Container,
+  NameContainer,
+  NameCryp,
+  NameSym,
+  LogoContainer,
+  Logo,
+  PriceContainer,
+  Price,
+  Percentage,
+  PreArrow,
+  ArrowD,
+  Separator,
+} from './style';
+import ArrowUp from '../../assets/arrowup.png';
+import ArrowDown from '../../assets/arrowdown.png';
+
+const Cryptoitems = ({item}: any) => {
```

Comment:
Don't use any type.
Create an interface for the props of the funcion 

interface Props {
   item: TYPE_OF_THE_ITEM
}

const Cryptoitems = ({item}: Props)